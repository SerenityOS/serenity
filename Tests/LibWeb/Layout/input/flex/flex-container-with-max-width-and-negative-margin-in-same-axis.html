<!DOCTYPE html><html><head><style>
#container-of-flex {
    max-width: 100px;
}

#flex {
    display: flex;
    margin-left: -100px;
    background-color: orange;
    height: 100px;
}
</style></head><body><div id="container-of-flex"><div id="flex"> </div>