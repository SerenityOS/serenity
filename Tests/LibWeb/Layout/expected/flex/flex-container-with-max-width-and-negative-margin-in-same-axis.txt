Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x116 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x100 children: not-inline
      BlockContainer <div#container-of-flex> at (8,8) content-size 100x100 children: not-inline
        Box <div#flex> at (-92,8) content-size 200x100 flex-container(row) [FFC] children: not-inline
          BlockContainer <(anonymous)> at (-92,8) content-size 0x0 [BFC] children: inline
            TextNode <#text>
